eclass-writing/text.xml

```diff
@@ -667,7 +667,7 @@ src_compile() {
 <body>
 
 <p>
-A simple eclass which defines a number of functions and a default
+A simple eclass which defines a number of functions and default <c>src_confgure</c> and
 <c>src_compile</c> for the (hypothetical) <c>jmake</c> build system might look
 something like the following:
```

Comment:
I think this paragraph could be better, I could rework it a bit.

Reply:
What are you thinking?

Reply:
Yeah, let's throw that in. Separate commit.

It reads easier :)

Reply:
Done.